# Copyright 2021 Ciena Corporation
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
/Users/dbainbri/src/multi-cluster/kubefed
diff --git a/scripts/fix-joined-kind-clusters.sh b/scripts/fix-joined-kind-clusters.sh
index cb675ad..158fca1 100755
--- a/scripts/fix-joined-kind-clusters.sh
+++ b/scripts/fix-joined-kind-clusters.sh
@@ -30,11 +30,6 @@ set -o pipefail
 # components run in-memory by local e2e tests.
 LOCAL_TESTING="${LOCAL_TESTING:-}"
 
-if [ "`uname`" != 'Darwin' ]; then
-  >&2 echo "This script is only intended for use on MacOS"
-  exit 1
-fi
-
 NS="${KUBEFED_NAMESPACE:-kube-federation-system}"
 
 INSPECT_PATH='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}'
